packages/overlay/src/integrations/sentry/utils/duration.ts

```diff
@@ -35,3 +35,18 @@ export function getFormattedDuration(duration: number): string {
 export function getFormattedSpanDuration(span: { timestamp: number; start_timestamp: number }): string {
   return getFormattedDuration(span.timestamp - span.start_timestamp);
 }
+
+export function formatTimestamp(timestamp: number) {
```

Comment:
Can we just stick to ISO-8601 format instead of US formatting? Pretty sure the rest of the world will thank us later on along with some US folks themselves.